.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+on: push
```

Comment:
Is this file used for CICD?

Reply:
Yup, github actions picks up anyfile under .github/workflows dir